graphics/src/backend.rs

```diff
@@ -6,6 +6,13 @@ use iced_core::{Font, Point, Size};
 
 use std::borrow::Cow;
 
+/// The graphics backend of a [`Renderer`].
+///
+/// [`Renderer`]: crate::Renderer
+pub trait Backend {
+    type Primitive;
```

Comment:
Not sure if we want documentation on this associated type for those implementing their own `Backend`